bridges/AppleAppStoreBridge.php

```diff
@@ -37,6 +37,19 @@ class AppleAppStoreBridge extends BridgeAbstract
                 'India' => 'IN',
                 'Canada' => 'CA',
                 'Germany' => 'DE',
+                'Netherlands' => 'NL',
+                'Belgium (NL)' => 'BENL',
+                'Belgium (FR)' => 'BEFR',
+                'France' => 'FR',
+                'Italy' => 'IT',
+                'United Kingdom' => 'UK',
+                'Spain' => 'ES',
+                'Portugal' => 'PT',
+                'Australia' => 'AU',
+                'New Zealand' => 'NZ',
+                'Indonesia' => 'ID',
+                'Brasil' => 'BR',
```

Comment:
Shouldn't "Brasil" be "Brazil"? Don't know whether that's how the AppleAppStore spells it though.

Reply:
You're right! Let me try changing it.

Reply:
@DRogueRonin Fixed